tests/DPDK_Flow_Dump_with_LNT.py

```diff
@@ -0,0 +1,204 @@
+        self.link_port_list = gnmi_ctl_utils.get_link_port_list(self.config_data)
+        log.info(self.gnmictl_hotplug_params)
+        self.interface_ip_list = get_interface_ipv4_dict(self.config_data)
+        self.conn_obj_list = []
+
+    def runTest(self):
+        log.info(f"Begin to configure OVVS and VM on local host")
+        if not gnmi_ctl_utils.gnmi_ctl_set_and_verify(self.gnmictl_params):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail("Failed to configure gnmi ctl ports")
+       
+        # Bring up control TAP1
+        if not gnmi_ctl_utils.ip_set_dev_up(self.link_port_list[0]["control-port"]):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail("Failed to bring up {self.link_port_list[0]['control-port']")
+
+        if not gnmi_ctl_utils.ip_add_addr(
+            self.link_port_list[0]["control-port"],
+            self.config_data["vxlan"]["tep_ip"][0],
+        ):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail(
+                "Failed to configure TEP IP for {self.link_port_list[0]['control-port']}"
+            )
+
+        # generate p4c artifact
+        if not test_utils.gen_dep_files_p4c_dpdk_pna_tdi_pipeline_builder(
+            self.config_data
+        ):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail("Failed to generate P4C artifacts or pb.bin")
+            
+        # set pipe line
+        if not p4rt_ctl.p4rt_ctl_set_pipe(
+            self.config_data["switch"],
+            self.config_data["pb_bin"],
+            self.config_data["p4_info"],
+        ):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail("Failed to set pipe")
+
+        # add a bridge to ovs
+        if not ovs_utils.add_bridge_to_ovs(self.config_data["bridge"]):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail(f"Failed to add bridge {self.config_data['bridge']} to ovs")
+        # bring up bridge
+        if not gnmi_ctl_utils.ip_set_dev_up(self.config_data["bridge"]):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail(f"Failed to bring up {self.config_data['bridge']}")
+
+        log.info(f"Config VXLAN Port on local host With Remote Tunnel IP")
+        if not ovs_utils.add_vxlan_port_to_ovs(
+            self.config_data["bridge"],
+            self.config_data["vxlan"]["vxlan_name"][0],
+            self.config_data["vxlan"]["tep_ip"][0].split("/")[0],
+            self.config_data["vxlan"]["tep_ip"][1].split("/")[0],
+            self.config_data["vxlan"]["dst_port"][0],
+        ):
+
+            self.result.addFailure(self, sys.exc_info())
+            self.fail(
+                f"Failed to add vxlan {self.config_data['vxlan']['vxlan_name'][0]} to bridge {self.config_data['bridge']}"
+            )
+
+        for i in range(len(self.conn_obj_list)):
```

Comment:
where are you populating conn_obj_list, its set to [] in setup(), please describe the purpose of this variable, maybe in setup()